src/main/java/com/cosium/code/format/InstallHooksMojo.java

```diff
@@ -53,6 +52,10 @@
   @Parameter(property = "debug", defaultValue = "false")
   private boolean debug;
 
+  /** Make the pre-commit hook quiet */
+  @Parameter(property = "quiet", defaultValue = "false")
```

Comment:
I think a more specific name would be better here.
Since this parameter is only expected to silent the hook, I propose `silentPreCommitHook`.